bundler.d/assets.rb

```diff
@@ -1,7 +1,8 @@
 gem 'jquery-rails', '~> 3.1'
 gem 'jquery-turbolinks', '~> 2.1'
 gem 'jquery-ui-rails', '< 5.0.0'
-gem 'bootstrap-sass', '3.0.3.0'
+gem 'bootstrap-sass'
+gem 'patternfly-sass'
```

Comment:
`patternfly-sass` relies on `bootstrap-sass`, feel free to remove the latter from here. Also please pin it to a stable version, `~> 2.7` should be fine